src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private final int[] numbers = new int[12];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = -1;
+
+    public int[] getNumbers() {
+        return numbers;
+    }
+
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == -1;
+    }
+
+    public boolean isFull() {
+        return total == 11;
+    }
+
+    protected int peekaboo() {
+        if (callCheck())
+            return -1;
+        return numbers[total];
+    }
+
+    public int countOut() {
+        if (callCheck())
+            return -1;
```

Comment:
Laco of extracting constants. Have you used review hints ('Github - PR checkpoint' at wikamp).